showcase/package.json

```diff
@@ -125,9 +125,9 @@
     "@angular/platform-server": "~7.2.2",
     "@angular/router": "~7.2.2",
     "@nationalbankbelgium/code-style": "^1.0.1",
-    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-rc.2-0c7801b2.tgz",
```

Comment:
Could you please revert the changes in this file since they are not necessary ? 😊